extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -435,18 +435,40 @@ public void startSearch(List<String> serialNumbers) {
         }
     }
 
+    /**
+     * Because the State can produce NPEs on getColorMode() and getAlertMode(), at first we check for the common
```

Comment:
You have written getAlertMode() could trigger a NPE.
Your implementation seems that you mean getEffect().

Or do I miss something here.